app/controllers/users_controller.rb

```diff
@@ -0,0 +1,48 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show]
+
+  # GET /users
+  # GET /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1
+  # GET /users/1.json
+  def show; end
+
```

Comment:
lists events created by the user, i.e. 
`@created_events = current_user.events` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

lists events previously attended by the user using event model scope, i.e. 
`@past_events = current_user.attended_events.past_events`

lists upcoming events previously attended/invited by the user using event model scope, i.e. 
`@upcoming_events = current_user.attended_events.upcoming_events`